swagger/swagger.yaml

```diff
@@ -355,6 +359,87 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+  '/files/{fileId}':
+    get:
+      summary: Get files details and all related scans results
+      description: |
+        Get detailled information about the given file. List all known scan
+        results for this file
+      tags:
+        - Files
+      parameters:
+        - name: fileId
+          in: path
+          type: string
+          description: ID of the file
+          required: true
+        - name: name
+          in: query
+          description: submitted file name
+          type: string
+        - name: offset
+          in: query
+          description: Offset the list of returned objects. Default is zero.
+          type: integer
+          format: int64
+        - name: limit
+          in: query
+          description: A limit on the number of objects to be returned.
+          type: integer
+          format: int64
+      responses:
+        '200':
+          description: OK
+          schema:
+            type: object
+            properties:
+              total:
+                type: integer
+                format: int64
+              offset:
+                type: integer
+                format: int64
+              limit:
+                type: integer
+                format: int64
+              data:
+                type: array
+                items:
+                  $ref: '#/definitions/SummaryScanResult'
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+  '/files/{fileId}/download':
+    get:
+      summary: Download file
+      description: |
+        Download an existing file sample
+      tags:
+        - Files
+      parameters:
+        - name: fileId
+          in: path
+          type: string
+          description: ID of the file
+          required: true
+      responses:
+        '200':
+          description: OK
+          schema:
+            type: file
+          headers:
+            Content-type:
+              type: string
+              description: application/octet-stream; charset=UTF-8
+            Content-Disposition:
+              type: string
+              description: attachment; filename=sha256 of the file
+          description: Export data into a file
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
 
 definitions:
```

Comment:
I haven't done that on the file before, but maybe we should organize definitions properties using Alphabetizer order. (Only if you've got 5 minutes).